sys/include/crypto/modes/cbc.h

```diff
@@ -36,9 +37,13 @@ extern "C" {
  * @param input_len  length of the input data
  * @param output     pointer to allocated memory for encrypted data. It has to
  *                   be of size data_len + BLOCK_SIZE - data_len % BLOCK_SIZE.
+ *
+ * @return            <0 on error
```

Comment:
Looking at the code, I would give the return code (e.g. CIPHER_ERR_INVALID_LENGTH and CIPHER_ERR_ENC_FAILED) returned to be more precise.